lib/checkNewCronJobs.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  serverJobAdmins
+} = require('../userWhitelist.json');
+const {
+  DATASTORE_LABEL,
+  getAllChangedFiles,
+  hasDatastoreLabel,
+  pingAndAssignUsers
+} = require('./utils');
+const CRON_REGISTRY_FILENAME = 'main_cron.py';
+const CRON_TEST_FILENAME = 'core/controllers/cron_test.py';
+const CRON_JOB_FILE = 'core/controllers/cron.py';
+
+const cronFileIsChanged = (changedFiles) => {
+  changedFiles.filter((file) => {
+    if (file.filename === CRON_JOB_FILE) {
+      return true;
+    }
+  });
+};
```

Comment:
I don't think this works as expected.
Have you tested that this works?

Reply:
What could be other possible fix to check that any changes have been made to `CRON_JOB_FILE`?

Reply:
I have one more doubt how is `action_build/index.js`  working
![Screenshot from 2021-03-22 15-36-36](https://user-images.githubusercontent.com/60519303/111973411-769cc080-8b24-11eb-9707-9406da6f6a13.png)
![Screenshot from 2021-03-22 15-36-53](https://user-images.githubusercontent.com/60519303/111973417-78668400-8b24-11eb-9b46-d9a69424307f.png)

Why doesn't it contain the cron-job-check I added?

Reply:
I tried console.log inside the if statement it returns true but it is unable to get the comment body
![Screenshot from 2021-03-22 16-09-28](https://user-images.githubusercontent.com/60519303/111977879-4572bf00-8b29-11eb-822d-50bebe854f97.png)